### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by tuxcell**
The </body> tag in index.html was missing

__Which issue does this PR address?__

fix(properties-panel-extension): add closing tag.

__Acceptance Criteria__

* [ ] Corresponds to the concept 
* [ ] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/.github/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
Thanks! Can you update your commit to [fit our message guideline](https://www.conventionalcommits.org/)?

File: properties-panel-extension/app/index.html [hunk 1]
```diff
@@ -48,4 +48,5 @@
   </ul>
 
   <script src="index.js"></script>
+  </body>
```
Comment (philippfromme) [inline properties-panel-extension/app/index.html:51]:
```suggestion
</body>
```

**Review by philippfromme — CHANGES_REQUESTED** [review]
Thanks for the PR. Let's fix the indentation so we can merge this.

**Comment by philippfromme** [comment]
Last thing: Let's adjust the commit message to `fix(properties-panel-extension): add closing tag`.

**Comment by tuxcell** [comment]
> Last thing: Let's adjust the commit message to `fix(properties-panel-extension): add closing tag`.

Done

**Comment by pinussilvestrus** [comment]
Your commit message(s) still have the wrong format: https://github.com/bpmn-io/bpmn-js-examples/pull/119/commits. Also, please [squash them together](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git) to one single commit.

**Pull request closed without merging**